setup.py

```diff
@@ -16,12 +19,21 @@
     "pytz",
     "python-dateutil<3.0.0,>=2.1",
     "mock",
+    "docker>=2.5.1"
```

Comment:
Forgive my ignorance but I don't see any `import docker` or `from docker import ...` lines in this PR. How is the `docker` library used?

Reply:
see https://github.com/spulec/moto/pull/1048/files#diff-2df06bdd95e3c78b9335da9f260a3472R6

Reply:
Thanks for helping me understand!